style.css

```diff
@@ -571,18 +583,604 @@ footer h6 {
+    justify-content: center;
+    align-items: center;
+    font-size: 12px;
+  }
+
+  #list-of-project6 {
+    display: none;
+  }
+
+  .project-list-2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 120px;
+    padding: 20px 20px;
+    text-align: center;
+    margin: 0.2em;
+    font-size: 12px;
+    background-color: #98dede;
+    color: #1a2236;
+  }
+
+  .project-list-3 {
+    display: none;
+  }
+
+  .project-list-4 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0.2em;
+    width: 54px;
+    height: 12px;
+    padding: 20px 40px;
+    font-size: 12px;
+    text-align: center;
+    background-color: #98dede;
+    color: #1a2236;
+  }
+
+  .contact-form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    flex-direction: column;
+    width: 100%;
+    height: auto;
+    place-items: center;
+    position: relative;
+    top: 16000;
+  }
+
+  .contact-form p {
+    width: 40%;
+    height: 100px;
+    color: #1a2236;
+    align-self: center;
+    text-align: center;
+    margin: 10px 30%;
+    font-size: 16px;
+    font-weight: 400;
+  }
+
+  #mobile-img {
+    display: none;
+  }
+
+  .form-dir {
+    display: flex;
+    gap: 10px;
+  }
+
+  label {
+    display: grid;
+    grid-template-rows: repeat(1fr);
+  }
+
+  label > span {
+    transition: 2s;
+    transform-origin: top left;
+  }
+
+  label > input[placeholder=" "]:not(:focus):placeholder-shown + span {
+    transform: translateY(1em) scale(1.25);
+    pointer-events: none;
+    opacity: 0.5;
+    position: relative;
+    left: 180px;
+  }
+
+  input {
+    border: 1px solid #1a2236;
+    width: 254px;
+    height: 60px;
+    border-radius: 4px;
+    display: flex;
+  }
+
+  .input-field {
+    width: 254px;
+    height: 60px;
+    border-radius: 4px;
+    display: flex;
+  }
+
+  textarea {
+    width: 100%;
+  }
+
+  #teal-img {
+    display: block;
+    margin-left: 40px;
+    margin-top: 20px;
+  }
+
+  #cross-img {
+    display: none;
+  }
+
+  #bg-contact-img1 {
+    display: block;
+    position: absolute;
+    left: 0;
+    bottom: 10px;
+    margin-top: 20px;
+  }
+
+  #btn-img {
+    display: block;
+    position: absolute;
+    right: 0;
+    bottom: 0;
+  }
+
+  #bg-contact-img {
+    display: none;
+  }
+
+  .about-section {
+    width: 100%;
+    margin-top: 120px;
+  }
+
+  .about-section-right {
+    width: 40%;
+    height: auto;
+    float: right;
+    position: absolute;
+    right: 5%;
+    margin-top: 150px;
+  }
+
+  #resume-img {
+    display: none;
+  }
+
+  .about-section-left {
+    width: 40%;
+    height: auto;
+    float: left;
+    position: relative;
+    left: 5%;
+    margin-top: 150px;
+  }
+
+  #about-section-list {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .about-section-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 0;
+  }
```

Comment:
- [ ] Your About section is being overlapped by the languages/frameworks/skills container in some screen sizes. Kindly fix it.
- [OPTIONAL] Try to use the same font-family and font-size for your title like in the Figma template.
![image](https://user-images.githubusercontent.com/61919617/172935821-662cf445-b97a-464c-8258-9637e5e760fc.png)
![image](https://user-images.githubusercontent.com/61919617/172935865-b45fa15c-a697-443c-aee3-ed78b17cfd7b.png)